lib/slax/poker/estimates.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Slax.Poker.Estimates do
+  use Slax.Context
+  alias Slax.Poker.Estimate
+
+  @valid_estimates [1, 2, 3, 5, 8, 13, 666]
```

Comment:
lol at 666

Reply:
Haha that's current functionality but now that you point it out I don't think we're adding the REJECTED label when this comes up